### PR TITLE
Fixes orbit stack overflow (again)

### PR DESCRIPTION
**Pull request by AnturK**
Or we could just stop this nonsense of orbiting each other earlier.

### CODE REVIEW

File: code/modules/orbit/orbit.dm [hunk 1]
```diff
@@ -43,6 +43,10 @@
 	if (!orbiter.orbiting) //admin wants to stop the orbit.
 		orbiter.orbiting = src //set it back to us first
 		orbiter.stop_orbit()
+	var/atom/movable/AM = orbiting
+	if(istype(AM) && AM.orbiting.orbiting == orbiter)
```
Comment (Cyberboss) [inline code/modules/orbit/orbit.dm]: is the istype necessary?

Comment (AnturK) [reply code/modules/orbit/orbit.dm]: Only movables can orbit but you can orbit turfs too.

Comment (Cyberboss) [reply code/modules/orbit/orbit.dm]: ew

**Review by Cyberboss — APPROVED** [review]

**Comment by Chnkr** [comment]
Good job, you broke it

**Comment by Chnkr** [comment]
Good job, you broke it

**Comment by Chnkr** [comment]
Good job, you broke it

**Pull request merged**